test_runner/src/main/kotlin/ftl/adapter/GcStorageExists.kt

```diff
@@ -0,0 +1,10 @@
+package ftl.adapter
+
+import ftl.adapter.google.gcStorageExist
+import ftl.api.RemoteStorage
+
+object GcStorageExists :
+    RemoteStorage.Exist,
+    (RemoteStorage.Dir) -> Boolean by {
+        gcStorageExist(it.bucket, it.path)
```

Comment:
@Sloox  this function should be placed under `client.google` package

Reply:
done